Tools/CISupport/ews-build/steps.py

```diff
@@ -140,6 +140,8 @@ class GitHubMixin(object):
     pr_open_states = ['open']
     pr_closed_states = ['closed']
     BLOCKED_LABEL = 'merging-blocked'
+    MERGE_QUEUE_LABEL = 'merge-queue'
```

Comment:
I think it's better to have these labels as a verb/action.

Reply:
Some other ideas:
`merge`/`merge-quickly`
`land`/`land-quickly`
`start-merge`/`start-fast-merge`